projects/plugins/boost/app/assets/src/js/stores/config.ts

```diff
@@ -22,6 +23,18 @@ export const dismissedPopOuts = {
 	},
 };
 
+// eslint-disable-next-line camelcase
```

Comment:
Is this needed? I don't see any non-camel case on the next line :)

Reply:
True, removed